CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+* **September 26 2023**
+  * Version 3.0.0
+  * jDataView no longer polyfills DataView, it uses the native implementation. This should make it faster for some operations.
```

Comment:
This claim seems dubious - didn't jDataView already use DataView when available? If so, how does removing fallbacks make it faster? I'd rather rephrase this to say that the size of the bundle is smaller or something.

Reply:
Yeah, I think I misunderstood some of how it used to work. Changed!